app/src/main/java/com/amaze/filemanager/ui/dialogs/GeneralDialogCreation.java

```diff
@@ -149,6 +149,25 @@ public static MaterialDialog showBasicDialog(
     return a.build();
   }
 
+  public static MaterialDialog createAlertDialog(
```

Comment:
Please move this to a kt object in dialogs package, annotated with `@JvmStatic`. Like dialogs in  #3205.